includes/wc-core-functions.php

```diff
@@ -1007,7 +1007,7 @@ function wc_print_js() {
 		$wc_queued_js = preg_replace( '/&#(x)?0*(?(1)27|39);?/i', "'", $wc_queued_js );
 		$wc_queued_js = str_replace( "\r", '', $wc_queued_js );
 
-		$js = "<!-- WooCommerce JavaScript -->\n<script type=\"text/javascript\">\njQuery(function($) { $wc_queued_js });\n</script>\n";
+		$js = "<!-- WooCommerce JavaScript -->\n<script type=\"text/javascript\" defer>\n$wc_queued_js\n</script>\n";
```

Comment:
Removing it will break any existing 3rd party plugin that relies on it.

Reply:
Sorry, I understand what you are saying, yet this will cause issues for existing plugins and integrations.